addons/point_of_sale/models/account_move.py

```diff
@@ -58,6 +58,10 @@ def _compute_payments_widget_reconciled_info(self):
                             'pos_payment_name': pos_payment.payment_method_id.name,
                         })
 
+    def get_invoice_pdf_report_attachment(self):
+        if self.env.user.has_group('point_of_sale.group_pos_user') and self.sudo().pos_order_ids:
```

Comment:
will an `hr.employee` user be able to create invoices?

Reply:
Do you mean someone with only `hr.employee` rights ?